styles/components/_modal.scss

```diff
@@ -0,0 +1,42 @@
+//MODAL
+.modal {
+  // Positioning /
+  position: absolute;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  // Display & Box Model /
+  display: flex;
+  width: 100%;
+  height: 100%;
+  align-items: center;
+  justify-content: center;
+  // Color /
+  background-color: rgba(0, 0, 0, 0.8);
+  &__container {
+    // Display & Box Model /
+    display: flex;
+    width: 80%;
+    height: 80%;
+    flex-direction: column;
+    justify-content: space-between;
+    padding: 48px;
```

Comment:
Can you use here more flexible units?

Reply:
I think it should be defined after we get the layout confirmation from the client. Right now these are to show how it behaves.